modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx

```diff
@@ -106,6 +110,35 @@ export default function EditObjectFolder({
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [showChangesSaved]);
 
+	const exportAsPDF = (
+		setExportingPDF: Dispatch<SetStateAction<boolean>>
+	) => {
+		setExportingPDF(true);
+
+		fitView({includeHiddenNodes: true, padding: 0.2});
+
+		const title = selectedObjectFolder.label[defaultLanguageId];
+
+		setTimeout(() => {
+			getPDF({
+				htmlElement: document.querySelector(
+					'.react-flow__renderer'
+				) as HTMLElement,
+				invertScale: getInvertScale(
+					document.querySelector('.react-flow__nodes')!
+				),
+				title,
+			}).then((pdf) => {
+				pdf.save(
+					title
+						? `${title.replace(/ /g, '_')}-Folder.pdf`
+						: 'Folder.pdf'
+				);
+				setExportingPDF(false);
+			});
+		}, 3000);
```

Comment:
Hey @cgmonte, do we really need 3s in this timeout? Would it be possible to reduce this time to 1s or something like that?